Snowplow/Internal/Entities/SPDeepLinkEntity.m

```diff
@@ -0,0 +1,57 @@
+//
+// SPDeepLinkEntity.m
+// Snowplow
+//
+// Copyright (c) 2013-2021 Snowplow Analytics Ltd. All rights reserved.
+//
+// This program is licensed to you under the Apache License Version 2.0,
+// and you may not use this file except in compliance with the Apache License
+// Version 2.0. You may obtain a copy of the Apache License Version 2.0 at
+// http://www.apache.org/licenses/LICENSE-2.0.
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the Apache License Version 2.0 is distributed on
+// an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the Apache License Version 2.0 for the specific
+// language governing permissions and limitations there under.
+//
+// Copyright: Copyright © 2020 Snowplow Analytics.
+// License: Apache License Version 2.0
+//
+
+#import "SPDeepLinkEntity.h"
+
+#import "SPUtilities.h"
+#import "SPPayload.h"
+#import "SPSelfDescribingJson.h"
+
+@interface SPDeepLinkEntity ()
+@property (nonatomic, nonnull) NSMutableDictionary<NSString *, NSObject *> *parameters;
+@end
+
+@implementation SPDeepLinkEntity
+
+NSString * const kSPDeepLinkSchema = @"iglu:com.snowplowanalytics.mobile/deeplink/jsonschema/1-0-0";
```

Comment:
Deep link is two words, no? Should this be `deep_link`?

Reply:
yeah, I was thinking the same preparing the PR for the schemas!